workspaces/arborist/test/query-selector-all.js

```diff
@@ -513,7 +514,7 @@ t.test('query-selector-all', async t => {
     ['*:has(* > #bar:semver(1.4.0))', ['foo@2.2.2']],
     ['*:has(> #bar:semver(1.4.0))', ['foo@2.2.2']],
     ['.workspace:has(> * > #lorem)', ['a@1.0.0']],
-    ['.workspace:has(* #lorem, ~ #b)', ['a@1.0.0']],
+    ['.workspace:has(* #lorem, ~ #b)', ['a@1.0.0', 'b@1.0.0']],
```

Comment:
The expected result for this test case was incorrect. Given that `a` is a dependency of `b`, `b` is ALSO a workspace with a dependency on `lorem` (via `a`).

The `~ #b` bit is sorta confusing here, but since `,` behaves like an OR I think it's still correct to include `b` in the result set (it's the `* #lorem` bit that should be selecting `b`, NOT the `~ #b` bit)